src/Trakx.Circle.ApiClient.Tests/Integration/Config.cs

```diff
@@ -0,0 +1,15 @@
+
+
+
+
+namespace Trakx.Circle.ApiClient.Tests.Integration
+{
+    public static class Config
+    {
+        /// <summary>
+        /// Todo Move to AWS Parameters
+        /// </summary>
+        public static  readonly string API_KEY = "QVBJX0tFWTo1ZWNlODI3MmRkZThiYWFjZDAxMzk1ZDViY2VhZTg4ZDo5MTZmY2YyYWJhMWU3YTk2MDM1OTVhMjQ5NDZjYTM4Mg==";
```

Comment:
No! We cannot store API key in the code at any point. Even if these are demo / sandbox keys.
This is the whole point of having the `Secrets` class or of fetching the configuration from AWS.

We don't do it for dev secrets, because although it has no consequences, it is very bad practice, and will one day give way to storing a production api key in a public repo.